style.css

```diff
@@ -0,0 +1,167 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+html {
+  overflow: hidden;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.intro {
+  background-image: url(./images/pic1.png);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  padding: 20px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+li {
+  list-style: none;
+}
+
+.burger {
+  display: none;
+}
+
+.welcome {
+  color: black;
+  display: none;
+}
+
+.nav-list {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  gap: 30px;
+  cursor: pointer;
+  font-size: 17px;
+}
+
+.nav-item {
+  color: black;
+}
+
+.page-links {
+  display: flex;
+  height: 20px;
+  justify-content: center;
+  padding-top: 20px;
+  gap: 20px;
+  cursor: pointer;
+}
+
+.icon-list {
+  color: black;
+}
+
+.content {
+  padding: 20px;
+}
+
+.greetings {
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 60px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+  padding-top: 100px;
+}
+
+.first {
+  color: green;
+  font-size: 36px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 32px;
+  font-style: normal;
+}
+
+.second {
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+  flex-wrap: wrap;
+  width: 50%;
+  margin: 0 auto;
+  margin-top: 25px;
+}
+
+/* media queries */
+@media screen and (max-width: 600px) {
+  .nav-list {
+    display: none;
+  }
+
+  .burger {
+    float: right;
+  }
+
+  .welcome,
+  .burger {
+    color: #000;
+    font-size: 20px;
+    padding: 10px;
+    display: inline-block;
+  }
+
+  .intro {
+    background-image: url(./images/pic1.png);
+    background-size: cover;
+    width: 475px;
+    height: 600px;
+  }
+
+  .greetings {
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 530;
+    line-height: 52px;
+    letter-spacing: 0;
+    text-align: center;
+    padding-top: 0;
+  }
+
+  .first {
+    font-size: 40px;
+    font-weight: 500;
+    line-height: 52px;
+    letter-spacing: 0;
+    text-align: center;
+  }
+
+  .second {
+    font-size: 20px;
+    font-weight: 400;
+    align-items: center;
+    line-height: 20px;
+    font-style: normal;
+    letter-spacing: 0;
+    width: 327px;
+    height: 164px;
+  }
+
+  .page-links {
+    cursor: pointer;
+  }
+
+  .welcome:hover {
+    text-decoration: underline;
+  }
+}
```

Comment:
- [ ] [OPTIONAL] It's amazing to see you write a CSS media query. Just I like said above, it was not necessary for this task. You will be required later on to work on CSS media queries for a desktop version. Working on it now is not a good idea.